index.d.ts

```diff
@@ -471,6 +499,35 @@ declare namespace Moleculer {
 		}
 	}
 
+	class Transit {
+		constructor(broker: ServiceBroker, transporter: Transporter, opts: GenericObject);
+		afterConnect(wasReconnect: boolean): Promise<void>;
+		connect(): Promise<void>;
+		disconnect(): Promise<void>;
+		sendDisconnectPacket(): Promise<void>;
+		makeSubscriptions(): Promise<Array<void>>;
+		messageHandler(cmd: string, msg: GenericObject): boolean | Promise<void> | undefined;
```

Comment:
The docs say this should be just a `boolean` but I see more possibilities in the code, not sure if this is correct or should the code be corrected to only output boolean

Reply:
It can be boolean or Promise